darshan-util/pydarshan/pyproject.toml

```diff
@@ -3,3 +3,24 @@ requires = [
     "wheel",
     "setuptools",
 ]
+
+[tool.cibuildwheel]
+before-all = [
+    "yum install -y blas-devel lapack-devel",
+    "yum install -y libxslt-devel libxml2-devel",
+    "yum install -y libjpeg libjpeg-devel",
+    "./prepare.sh",
+    "./configure --disable-darshan-runtime --enable-apxc-mod --enable-apmpi-mod",
+    "make install"
+]
+environment = "PYDARSHAN_BUILD_EXT=1"
+skip = [
+    "*musllinux*",
+    "*i686*"
+]
+test-requires = [
+    "pytest",
+    "lxml",
+    "importlib_resources;python_version<'3.9'"
+]
+test-command = "pytest {package}"
```

Comment:
may need a rebase to filter out these changes already present in `main`

Reply:
oops, my mistake. this is fixed now.